src/datastore/sync_methods/index.js

```diff
@@ -420,5 +420,28 @@ export default {
 
     // return resource from cache
     return resource.previousAttributes[id] ? DSUtils.copy(resource.previousAttributes[id]) : undefined;
+  },
+  /**
+   * Revert all attributes of the item with the given primary key th their previous values.
```

Comment:
`primary key th their previous values` should be `primary key to their previous values`